runtime/help/options.md

```diff
@@ -40,65 +40,65 @@ Here are the available options:
+   default value: `""` (empty string)
 
 * `basename`: in the infobar and tabbar, show only the basename of the file
    being edited rather than the full path.
 
-    default value: `false`
+   default value: `false`
 
 * `clipboard`: specifies how micro should access the system clipboard.
    Possible values are:
-    * `external`: accesses clipboard via an external tool, such as xclip/xsel
-       or wl-clipboard on Linux, pbcopy/pbpaste on MacOS, and system calls on
-       Windows. On Linux, if you do not have one of the tools installed, or if
-       they are not working, micro will throw an error and use an internal
-       clipboard.
-    * `terminal`: accesses the clipboard via your terminal emulator. Note that
-       there is limited support among terminal emulators for this feature
-       (called OSC 52). Terminals that are known to work are Kitty (enable
-       reading with `clipboard_control` setting), iTerm2 (only copying),
-       st, rxvt-unicode and xterm if enabled (see `> help copypaste` for
-       details). Note that Gnome-terminal does not support this feature. With
-       this setting, copy-paste **will** work over ssh. See `> help copypaste`
-       for details.
-    * `internal`: micro will use an internal clipboard.
-
-    default value: `external`
+   * `external`: accesses clipboard via an external tool, such as xclip/xsel
+      or wl-clipboard on Linux, pbcopy/pbpaste on MacOS, and system calls on
+      Windows. On Linux, if you do not have one of the tools installed, or if
+      they are not working, micro will throw an error and use an internal
+      clipboard.
+   * `terminal`: accesses the clipboard via your terminal emulator. Note that
+      there is limited support among terminal emulators for this feature
+      (called OSC 52). Terminals that are known to work are Kitty (enable
+      reading with `clipboard_control` setting), iTerm2 (only copying),
+      st, rxvt-unicode and xterm if enabled (see `> help copypaste` for
+      details). Note that Gnome-terminal does not support this feature. With
+      this setting, copy-paste **will** work over ssh. See `> help copypaste`
+      for details.
+   * `internal`: micro will use an internal clipboard.
+
+   default value: `external`
 
 * `colorcolumn`: if this is not set to 0, it will display a column at the
    specified column. This is useful if you want column 80 to be highlighted
    special for example.
 
-	default value: `0`
+   default value: `0`
 
-* `colorscheme`: loads the colorscheme stored in 
+* `colorscheme`: loads the colorscheme stored in
    $(configDir)/colorschemes/`option`.micro, This setting is `global only`.
 
-	default value: `default`
+   default value: `default`
 
-    Note that the default colorschemes (default, solarized, and solarized-tc)
-    are not located in configDir, because they are embedded in the micro
-    binary.
+   Note that the default colorschemes (default, solarized, and solarized-tc)
```

Comment:
> (Still, ideally in a separate commit, since this about fixing number of spaces, not about converting tabs to spaces. Again, imagine you're reviewing this.)

Yes, ideally. But in the moment doing the rework by going line over line through the file I look to the left and right and pick up the things that I don't like. I apologize for not holding back these changes and tracking them with their own commit and you can trust me that I know that such reviews are ugly.